CHANGELOG.md

```diff
@@ -6,11 +6,20 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Added
+
+- Now each system extension has its own weight, defined by `ExtensionWeightInfo` ([polkadot-fellows/runtimes/pull/605](https://github.com/polkadot-fellows/runtimes/pull/605))
+- Parachains define the default `CoreSelector` strategy, according to [`RFC-0103`](https://polkadot-fellows.github.io/RFCs/approved/0103-introduce-core-index-commitment.html) ([polkadot-fellows/runtimes/pull/605](https://github.com/polkadot-fellows/runtimes/pull/605))
+
 ### Changed
 
 - Disable MBM migrations for all runtimes for check-migrations CI ([polkadot-fellows/runtimes/pull/590](https://github.com/polkadot-fellows/runtimes/pull/590))
 - chain-spec-generator supports conditional building (`--no-default-features --features <runtime>` or `--no-default-features --features all-runtimes` or
   `--no-default-features --features all-polkadot` or `--no-default-features --features all-kusama`)([polkadot-fellows/runtimes/pull/637](https://github.com/polkadot-fellows/runtimes/pull/637))
+- Update to SDK version `2412-2` ([polkadot-fellows/runtimes/pull/605](https://github.com/polkadot-fellows/runtimes/pull/605))
+- Migration to XCMv5 ([polkadot-fellows/runtimes/pull/605](https://github.com/polkadot-fellows/runtimes/pull/605))
+- Changed from `SignedExtension` to `TransactionExtension` ([polkadot-fellows/runtimes/pull/605](https://github.com/polkadot-fellows/runtimes/pull/605))
+- `sp-std` removed from runtimes ([polkadot-fellows/runtimes/pull/605](https://github.com/polkadot-fellows/runtimes/pull/605))
```

Comment:
not relevant to users of the runtime/chain
```suggestion
```

Reply:
Regardless, we had to reintroduced it (temporarily 😢 )